cip/1.accepted/CIP2016-06-14-Define-comparability-and-equality-as-well-as-orderability-and-equivalence.adoc

```diff
@@ -434,7 +464,7 @@ SQL also treats comparisons involving `null` as returning `null`.
 
 PostgresSQL treats some numerical operations (such as division by zero) that would compute a `NaN` value as a numerical error that causes the query to fail.
 PostgresQL considers `NaN` values to be greater than positive infinity, both in comparison and in sort order.
-This proposal achieves something very similar by evaluating comparisons involving a `NaN` to `null` and by treating both `NaN` values as the largest numbers and `null` values as the largest values in the <<global-sort-order>>.
+This proposal achieves something very similar by treating both `NaN` values as the largest numbers and `null` values as the largest values in the <<global-sort-order>>.
```

Comment:
Simplifying this a little:
```suggestion
This proposal achieves something very similar by treating both `NaN` and `null` values as the largest values in the <<global-sort-order>>.
```

However, it's a bit strange to say that these are all 'largest'; do we see a sorted list of `null, NaN, null, NaN` as correct, even when interleaved? I guess I'm OK with that.

Reply:
Your suggestion is not what I meant. In 3.3.1 we list the global sort order between types, and see that `null` is ordered after `NaN`. What I meant is that `NaN` is the largest _number_ and `null` the largest overall _value_. Is that clear enough from what I wrote and you just over-read that subtlety or should I try to make it clearer?

Reply:
Yes you are right, I misread. Perhaps we can strike the word `both` from the sentence, as that caused me to believe it was an enumeration like 'by treating both X and Y as foo' which was my misunderstanding?

Reply:
Yes